tests/Doctrine/Migrations/Tests/TestLogger.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Doctrine\Migrations\Tests;
+
+use DateTime;
+use DateTimeInterface;
+use Psr\Log\AbstractLogger;
+use function get_class;
+use function gettype;
+use function is_object;
+use function is_scalar;
+use function method_exists;
+use function strpos;
+use function strtr;
+
+class TestLogger extends AbstractLogger
```

Comment:
Any reason why we can't use [`Psr\Log\Test \TestLogger`](https://github.com/php-fig/log/blob/446d54b4cb6bf489fc9d75f55843658e6f25d801/Psr/Log/Test/TestLogger.php#L57) here?

Copied from https://github.com/lcobucci/doctrine-migrations/pull/1/commits/37954e3c3790a53a819906e0ea5e22eddd3054ea

Reply:
Did not notice that. Will do!

Reply:
I checked better, the PSR test logger is not doing the string interpolation, that makes easier to test text messages.